packages/x-components/src/plugins/devtools/timeline.devtools.ts

```diff
@@ -0,0 +1,103 @@
+import { DevtoolsPluginApi } from '@vue/devtools-api';
```

Comment:
I don't see the benefit of using a class here 😅

Reply:
While I initially thought it might be a good refactor, it only complicates stuff.

To benefit from using a class, the class should be instantiated immediately as, the xplugin needs a reference to it, and at that point it does not have the devtools api, so you would end up with another "setup" method. There could be solutions like forcing the class to be a singleton, but it would be similar to this module. Moreover, I think it could bring some tree-shaking problems for production builds, so I would not fight this battle right now.